contracts/Bond.sol

```diff
@@ -289,12 +286,12 @@ contract Bond is
         // that would break in the case of a token taking a fee.
         // maybe we don't care about reentrency for this method? I was trying to think through potential exploits here, and
         // if reentrency is exploited here what can they do? Just pay over the maximum amount?
-        uint256 amountPaid = IERC20Metadata(paymentToken).safeTransferIn(
+        IERC20Metadata(paymentToken).safeTransferFrom(
             _msgSender(),
             address(this),
             amount
         );
-        emit Payment(_msgSender(), amountPaid);
```

Comment:
This seems like useful information to emit. If we're not going to be checking the actual amount, can we send the amount tried to transfer?